.gitignore

```diff
@@ -30,3 +30,4 @@ coverage/
 .tool-versions
 .rspec_status
 .rdbg_history
+.idea
```

Comment:
Is this intentional?

Reply:
It would seem the https://github.com/excon/excon/issues/800 is resolved in in version 0.95 onwards. When the version is fixed in https://github.com/dependabot/dependabot-core/commit/6d59a5ae80465db698e4cdc29270957ecd7bfc2c by preventing 0.94 onwards, I see the range has been expanded to 0.98 recently. See https://github.com/dependabot/dependabot-core/pull/6403. I run a whole update for a repository without any errors since I skipped that version 0.94.0 version with issues.

Will remove this the ignore.

Reply:
Yep, the issue seems already fixed, we just need to validate that upgrading causes no issues for us. We need to validate this on our internal systems, I will try to move that forward.

Reply:
Thanks. Looking forward to that

Reply:
Just let me know if you see any issues related to excon/excon#800 still lingering, it seemed like we squished the related issues, but they were a bit hard to pin down in the first place...

Reply:
We will, thanks for your prompt effort on fixing that ❤️